### PR TITLE
OSDOCS-478 updates to GCP machinesets

### PR DESCRIPTION
**Pull request by kalexand-rh**
Follow up to https://github.com/openshift/openshift-docs/pull/16787

@sunzhaohua2, PTAL?

### CODE REVIEW

File: modules/machineset-yaml-gcp.adoc [hunk 1]
```diff
@@ -59,9 +59,9 @@ spec:
...
           networkInterfaces:
-          - network: <infrastructureID>-network
-            subnetwork: <infrastructureID>-worker-subnet <1>
-          projectID: openshift-gce-devel
+          - network: <infrastructureID>-network <1>
+            subnetwork: <infrastructureID>-<role>-subnet <2>
+          projectID: <project_name> <4>
           region: us-central1
           serviceAccounts:
           - email: <infrastructureID>-w@openshift-gce-devel.iam.gserviceaccount.com <1>
```
Comment (sunzhaohua2) [inline modules/machineset-yaml-gcp.adoc]: - email: <infrastructureID>-w@<project_name>.iam.gserviceaccount.com <1> <4>

**Comment by kalexand-rh** [comment]
Thank you @sunzhaohua2! Please let me know if you see other changes that I need to make.

**Comment by sunzhaohua2** [comment]
LGTM, thank you.

**Pull request merged**